build-settings/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "build-settings",
+  "version": "0.0.1",
+  "main": "index.js",
+  "license": "MIT",
+  "private": true
```

Comment:
This isn't going to be published? Do we not having to prefix this with `@khanacademy`? What happens when we publish the Wonder Blocks packages that depend on this?

Reply:
When we go to install one of our `wonder-blocks-*` packages somewhere, this should be ignored b/c we've marked it as a dev dependency.  The `@khanacademy` prefix isn't necessary but we could add it if we wanted.  Since it's not a package we intend on publishing I opted not to include prefix to better differentiate it from those packages we are publishing.

Reply:
In this world if I `npm install wonder-blocks-foo` won't it get confused as it doesn't know about `build-settings`? Wouldn't it try to install it directly from NPM?

Reply:
Hmm - ok! I think a better name would be something like `wonder-blocks-build-settings` or some such, just to make it clear that this isn't some third-party package.

Reply:
How do you folks feel about using a naming convention that encapsulates the internal, dev-only nature of this style of package, making that more obvious?

`wb-dev-build-settings` or just `wb-dev-build`?

For some reason, it makes me a little uneasy how similar to our prod packages this looks.

Reply:
I'll change into `wb-dev-build`.